### ISSUE TITLE
Question: Will this also support IE11?

### ISSUE THREAD

**Issue by himhim1125** [ampproject/worker-dom#217]
I have tried to run the demo in local but find that most demos are not working well when using IE11

**Comment by kristoferbaxter** [comment]
Yes, this will support IE11. Most of the missing pieces at this point are listed as TODOs in the codebase.
